### ISSUE TITLE
/en-US/docs/Web/API/Canvas_API/Tutorial

### ISSUE THREAD

**Issue by SeasonLeee** [mdn/sprints#2235]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

In Canvas API Tutorial, the image of summary is covered with the border from the next section, as we can see below:
![image](https://user-images.githubusercontent.com/13610283/66717937-bda36400-ee10-11e9-96ba-9a0c0b9cb391.png)

and I have console log my screen's width... but I don't think this is the problem... and why I check the width is because when I resize the screen to a smaller ratio, it's fine, like this:
![image](https://user-images.githubusercontent.com/13610283/66718014-aa44c880-ee11-11e9-9879-b6ee029f30ba.png)

Ok, then I dived in to the code, realizing the image is floated, which is why the next pass through it.😅

Then I tried to fix this... I tried to use `display: flow-root` to the div that contains the image, then this will generate more issue like, it's not inline within the summary section, like this:
![image](https://user-images.githubusercontent.com/13610283/66718087-58507280-ee12-11e9-9ee2-8b9b68153a02.png)

So, without further experiment, we can know that clear fix won't work either, I think...

Then, the proper fix will be...
...
...

Why not use flexbox in the first place? 😅

**Comment by SeasonLeee** [comment]
Is the way I open the issue wrong? Or just mdn is run out of people to take care of documentation kind of issue? Well... I don't know how high the bar is, but I would love to give a hand...

**Comment by chrisdavidmills** [comment]
We have a huge backlog of work, and limited staff to take care of it all, so sometimes we are a bit slow replying to issues. Sorry about that.

If you fancy having a go at fixing this yourself, then please do ;-)

**Comment by SeasonLeee** [comment]
@chrisdavidmills haha, sorry! And I totally understand the case that you guys are running out of the people😅 And I would love to give this issue a try, as I have described above, I actually did! 

Now I wonder where's the real code of doc so I can have a peek of it to see if I can actually give it a shot. Or the repository that contains the doc?

Besides, I found another issue too... But that will be another story in another issue page😆

TL;DR, where's the actual doc code and where can I read the 'code of conduct' or the contribution guideline? Let me know so that I can pull up my sleeve and fix that bad boy😆

**Comment by chrisdavidmills** [comment]
I've had another look at this — there is not really an easy way to fix it.
